doc/source/ray-core/fault_tolerance/gcs.rst

```diff
@@ -0,0 +1,71 @@
+.. _fault-tolerance-gcs:
+
+GCS Fault Tolerance
+===================
+
+Global Control Service (GCS) is a server that manages cluster-level metadata.
+It also provides a handful of cluster-level operations including :ref:`actor <ray-remote-classes>`, :ref:`placement group <ray-placement-group-doc-ref>` and node management.
+By default, GCS is not fault tolerant since all the data is stored in-memory and its failure means that the entire Ray cluster fails.
+To make GCS fault tolerant, HA storage is required. Ray supports Redis as the backend storage for durability and high availability.
```

Comment:
`high availability.` do you mean the storage has to be HA?

Reply:
Good question. Should we mention HA? For FT, theoretically durability is enough?

Reply:
> Ray supports Redis as the backend storage for durability and high availability.

My confusing part is that whether durability and high availability is for Ray or Redis.
Like Redis needs to be durability and high availability. Or with redis Ray will be durability and high availability.